src/Task/Git/CommitMessage.php

```diff
@@ -56,6 +60,7 @@ public static function getConfigurableOptions(): OptionsResolver
         $resolver->addAllowedTypes('max_body_width', ['int']);
         $resolver->addAllowedTypes('max_subject_width', ['int']);
         $resolver->addAllowedTypes('case_insensitive', ['bool']);
+        $resolver->addAllowedTypes('skip_on_merge_commit', ['bool']);
```

Comment:
Can you add this one to the documentation of the task?
https://github.com/phpro/grumphp/blob/v2.x/doc/tasks/git_commit_message.md